examples/advanced/torque-controls.html

```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Animation | CARTO VL</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8">
+    <script src="../../dist/carto-vl.js"></script>
+    <script src="https://cartodb-libs.global.ssl.fastly.net/mapbox-gl/v0.44.1-carto1/mapbox-gl.js"></script>
+    <link href="https://api.tiles.mapbox.com/mapbox-gl-js/v0.44.1/mapbox-gl.css" rel="stylesheet" />
+    <link href="https://fonts.googleapis.com/css?family=Roboto:100,200,400,500" rel="stylesheet">
+    <style type="text/css">
+        * {
+            margin: 0;
+            padding: 0;
+        }
+
+        body {
+            display: flex;
+            height: 100vh;
+        }
+
+        #sidebar {
+            padding: 25px;
+            width: 275px;
+            background: #f8f8f8;
+        }
+
+        #sidebar h1 {
+            font: 100 42px/1.4 'Roboto';
+            margin-bottom: 10px;
+        }
+
+        #sidebar h2 {
+            font: 200 21px/1.4 'Roboto';
+            margin-bottom: 10px;
+        }
+
+        #sidebar div {
+            font: 300 14px/1.6 'Roboto';
+            margin-bottom: 20px;
+        }
+
+        #sidebar div input {
+            margin-right: 5px;
+        }
+
+        input[type="button"] {
+            align-items: center;
+            background: #1785FB;
+            border-radius: 4px;
+            border: 0;
+            box-shadow: none;
+            color: #fff;
+            cursor: pointer;
+            display: inline-flex;
+            font: 500 12px/20px 'Roboto';
+            margin: 0;
+            padding: 6px 12px;
+            transition: all 0.2s ease;
+            vertical-align: top;
+        }
+
+        #map {
+            flex: 1;
+        }
+    </style>
+</head>
+
+<body>
+    <div id="sidebar">
+        <h1>Torque</h1>
+        <h2>Controls</h2>
+        <div>
+            <input type="button" id="js-play-btn" value="PLAY">
+            <input type="button" id="js-stop-btn" value="STOP">
+            <input type="button" id="js-pause-btn" value="PAUSE">
+        </div>
+        <h2>Duration</h2>
+        <div>
+            <input type="range" id="js-duration-range" min="1" max="60" step="1">
+            <span id="js-duration-span"></span> 10 seconds
```

Comment:
The `10` should be within the span or things like `40 10 seconds` happens

Reply:
wops